WordPress/Classes/Stores/StatsInsightsStore.swift

```diff
@@ -0,0 +1,130 @@
+import Foundation
+import WordPressFlux
+import WordPressComStatsiOS
+
+enum InsightAction: Action {
+    case receivedLatestPostSummary(_ latestPostSummary: StatsLatestPostSummary?)
```

Comment:
There's some weird whitespace thing going on here

Reply:
Ummmm... I don't know what's going on. It's a valid underscore. There is no extra whitespace. I literally retyped the entire line and it made no difference.

Reply:
github gonna github i guess!